bakerydemo/static/css/main.css

```diff
@@ -385,6 +385,24 @@ li.has-submenu a.allow-toggle {
 .caret-custom:after {
     content: "▼"!important;
 }
+/* Mobile menu styling */
+@media (max-width: 768px) {
+  .nav-pills>li {
+    display: block;
+    float: none;
+    width: 100%!important;
+  }
+  .nav-pills> li > a {
+    border-top: 1px solid rgba(255,255,255,.1);
```

Comment:
As far as I can see this only changes the color, the border is already defined elsewhere – so this should only override `border-top-color`.